### ISSUE TITLE
awlsimhw_linuxcnc: Error on clicking "Add inspection: Online block"

### ISSUE THREAD

**Issue by uncle-yura** [mbuesch/awlsim#18]
An error occurs when the server passes an instance of hal to gui.
```
Traceback (most recent call last):
  File "/home/lcnc/awlsim/examples/linuxcnc-demo/../..//awlsim-linuxcnc-hal", line 438, in <module>
    sys.exit(main())
  File "/home/lcnc/awlsim/examples/linuxcnc-demo/../..//awlsim-linuxcnc-hal", line 394, in main
    server.run()
  File "/home/lcnc/awlsim/awlsim/coreserver/server.py", line 1691, in run
    self.__handleCommunication()		#@cy-win
  File "/home/lcnc/awlsim/awlsim/coreserver/server.py", line 1466, in __handleCommunication
    self.__handleSocketComm(rlist)
  File "/home/lcnc/awlsim/awlsim/coreserver/server.py", line 1439, in __handleSocketComm
    self.__handleClientComm(self.__sock2client[sock])
  File "/home/lcnc/awlsim/awlsim/coreserver/server.py", line 1423, in __handleClientComm
    handler(self, client, msg)
  File "/home/lcnc/awlsim/awlsim/coreserver/server.py", line 1340, in __rx_GET_IDENTS
    client.transceiver.send(reply)
  File "/home/lcnc/awlsim/awlsim/coreserver/messages.py", line 1918, in send
    data = memoryview(msg.toBytes())
  File "/home/lcnc/awlsim/awlsim/coreserver/messages.py", line 1582, in toBytes
    payload.append(self.packString(pVal))
  File "/home/lcnc/awlsim/awlsim/coreserver/messages.py", line 180, in packString
    return cls.packBytes(string.encode("utf-8", "strict"))
AttributeError: Pin 'encode' does not exist

```

In file "awlsim/coreserver/messages.py"  attached such a bone:

```
175: 	@classmethod
	def packString(cls, string):
		try:
			if not string:
				string = ""
			try:
				ret=cls.packBytes(string.encode("utf-8", "strict"))
			except:
				ret=cls.packBytes("")

			return ret 
		except UnicodeError as e:
			raise ValueError
```

**Comment by mbuesch** [comment]
Thanks for your bug report.

**Comment by mbuesch** [comment]
Fixed in Awlsim 0.72.0.
